csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -611,6 +600,17 @@ public override string ToString()
             }
         }
 
+        // May increase the size of the internal array, but will never shrink it.
```

Comment:
Any reason this has been moved rather than just changed from private to internal?

Reply:
(With my suggested approach of a new internal SetCount method, this could stay unchanged.)

Reply:
I moved it down as it appeared to be in the wrong place for an internal method. However, after reviewing the code again, I'm not sure if there even is any specific "ordering by visibility" of members in this class.